### PR TITLE
Add reputation labels

### PR DESCRIPTION
**Pull request by ErikSchierboom**
We want to allow awarding reputation to issues, like we're already doing with PRs.
Unfortunately, for PRs we're using `x:size/X` labels, but these have a distinct meaning for issues in that they describe the expected amount of work for that issue.
We thus came up with the idea to use a _separate_ set of labels: `x:rep/X`.
These labels can be added together with the size labels.
The `x:rep` labels also have the benefit of being more semantic than the `x:size` labels.

See https://github.com/exercism/exercism/issues/6318

Note: a separate PR (to be added later) should be merged together with this PR, which will automatically comment on the usage of `x:size` labels on PRs to point people to using `x:rep`.

### CODE REVIEW

**Review by SaschaMann — COMMENTED** [review]
Could you add the amount that's being rewarded to the description? Or better yet, the amount of working hours that corresponds to?

**Comment by ErikSchierboom** [comment]
> Or better yet, the amount of working hours that corresponds to?

Possibly

**Comment by ErikSchierboom** [comment]
> Could you add the amount that's being rewarded to the description? Or better yet, the amount of working hours that corresponds to?

We'd rather not add the amount being awarded to the description, as it depends on whether someone authored or reviewed it.

As for the amount of working hours. We discussed it, but have so far decided not to do it, as we want to give it a little more thought.

I'm merging the PR to get the PRs out to the repos to allow them to award reputation to issue authors.

**Review by iHiD — APPROVED** [review]

**Comment by SaschaMann** [comment]
> We'd rather not add the amount being awarded to the description, as it depends on whether someone authored or reviewed it.

Why not add both?

File: global-files/.github/labels.yml [hunk 1]
```diff
@@ -86,6 +86,30 @@
   description: "Work on Test Runners"
   color: "ffffff"
 
+# The `x:rep/<value>` labels describe the amount of reputation to award
+#
```
Comment (ee7) [inline global-files/.github/labels.yml:90]: Trailing whitespace on this line, FYI.

**Comment by iHiD** [comment]
To just colour this in a little more, for the time bit, the reason we're treading carefully is that adding more information to what's displayed will likely change the behaviours of people, and we want to think through those changes before we do them. For example, if we added the hours, it would then be easy to get a contributor who's taken 5hrs to do something that you only rate at a 30min PR and them feel frustration over that. Whereas if the it's purely a label, which in the docs groups PRs by example based on their relative size (as they currently do), we're not making judgement calls on whether someone is fast/slow, just on the impact of the PR to Exercism. If there was a way to show this information just to maintainers, but not also to contributors we'd add it now, but without that, I want some more time for us to consider the potential impacts before we add it.

For the amounts, I'd be more happy to do that and we could add both, but I'm not sure it's actually relevant as part of the decision-making criteria and the reason we gave labels, not numbers in the first place, was that so judgements would be made by categories, not by rep-counts. 

So TL;DR; We might decide to do both of these, but I want time to think about it all more before we do it.

File: global-files/.github/labels.yml (hunk 1 above)
Comment (SaschaMann) [reply global-files/.github/labels.yml:90]: Do you want to add that glorious configlet CI to this repo as well? :D

**Comment by SaschaMann** [comment]
That makes sense 👍

File: global-files/.github/labels.yml (hunk 1 above)
Comment (ee7) [reply global-files/.github/labels.yml:90]: OK, I can make a PR. 

The whitespace linting actions I found in the GitHub Marketplace seemed to be incomplete or have some flaw, so it was easier to just write:
https://github.com/exercism/configlet/blob/146d41c184b5/.github/bin/lint-whitespace

We could also consider adding it as an org-wide workflow. But this repo seems like a good first step.

Comment (SaschaMann) [reply global-files/.github/labels.yml:90]: Yeah, I think for now just having it here would help.

Comment (ee7) [reply global-files/.github/labels.yml:90]: > OK, I can make a PR.

Created #205
